app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class ArticlesController < ApplicationController
+  before_action :set_article, only: %i[show edit update destroy]
+
+  def index
+    @articles = Article.sorted.paginate(page: params[:page])
+  end
+
+  def show; end
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
```

Comment:
no need to use instance variable here.

Reply:
I don't think there is a need for changing it, in my opinion it is better to use instance variable

Reply:
agree